Apollo/Text/en/CodexText.en.sjson

```diff
@@ -1,23 +1,24 @@
 {
   /*
   Made by AlexKage and physiX
+  Additional writing by bessyboo
   */
   Texts = [
 	"_append"
     {
       Id = "CodexData_Apollo_0001"
       Speaker = "Achilles"
-      DisplayName = "'...One of Zeus's many sons, he is the god of archery and poetry, music and dance, truth and prohecy, healing and diseases...'"
+      DisplayName = "'...Much like his twin sister Artemis, Apollo is the god of archery and loves a good hunt. However, he is also the god of poetry, music and dance, truth and prohecy, even healing and diseases. But most notably of all, he is the god of the sun and its radiant light..."
```

Comment:
Not the god of the sun. God of light. This creates confusion with Helios. Let's try to not say Apollo is god of the sun since all others vanilla dialog doesn't seem to agree that Apollo is. If you could re-phrase to make it more God of light that would be great.

Reply:
Will do! That was actually part of what I kept from your originally-written codex entry, so it’s no skin off my nose to change it, lol